unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -77,6 +78,15 @@ public static UTF8String fromString(String str) {
     }
   }
 
+  /**
+   * Creates an UTF8String that contains `length` spaces.
+   */
+  public static UTF8String blankString(int length) {
+    byte[] spaces = new byte[length];
+    Arrays.fill(spaces, (byte) ' ');
```

Comment:
does ' ' always become space if utf8 encoding is not the default in the system? might be better to explicitly write down 0x20